### ISSUE TITLE
simulation freezes at start

### ISSUE THREAD

**Issue by hightower8083** [fbpic/fbpic#391]
There is a strange behaviour, which can be reproduced modifying the example script (_Standard simulation of laser-wakefield acceleration_), as follows:
- `n_order = 32`
- `Nr = 600`
- `p_zmin = zmin`
- `ramp_start = zmin`
- comment `#select={"uz" : [1., None ]},`

`mpirun -np 2` gets frozen on start (`Just-In-Time compilation`), more precisely, at writing `ParticleDiagnostic`. 

Tested for both `cuda=True` and `False`, on two different Linux machines. Skipping this first damp or reducing  `Nr` unblocks the issue.

**Comment by RemiLehe** [comment]
Thanks for reporting this issue!
I was able to reproduce it. It seems that this is an OpenMPI issue. (see [this post](https://stackoverflow.com/questions/54811518/mpi-gather-call-hangs-for-large-ish-arrays) for instance). 

In my case, I was able to fix it by forcing `conda` to install `mpi4py` with MPICH:
```
conda uninstall openmpi
conda install -c conda-forge mpi4py=*=*mpich*
```
Could you confirm that this works for you too? If yes, could you close the current issue?

**Comment by hightower8083** [comment]
yes, `mpich` fixes it, and also it works with `openmpi@3.1.4`, which I have from _spack_, in which case  `mpi4py` is installed via `pip`. Probably `openmpi@4.0.1` is still too raw..
